test/README.md

```diff
@@ -3,23 +3,18 @@
 ## Assumptions & Requirements
 
 1. Directory paths should be well formed such that they can be run from any directory.
-1. Test scripts in the `$HOME/test` directory assume they are run against a functional Openshift cluster that was setup prior to the execution of the test script.
-1. Each script in `$HOME/test` should has a companion script in `$HOME/hack/testing`
-1. Each script in `$HOME/test` performs cleanup logic to reset the state of the cluster and logging stack
+1. Test scripts in the `origin-aggregated-logging/test` directory assume they are run against a functional Openshift cluster that was setup prior to the execution of the test script.
+1. Each script in `origin-aggregated-logging/test` should has a companion script in `origin-aggregated-logging/hack/testing` which can be used to launch the test and do requisite setup steps.
```

Comment:
I don't think this is necessary.  I thought the `/hack/testing` scripts were only to allow us to run the `/test` scripts using `logging.sh`.